go.mod

```diff
@@ -7,8 +7,9 @@ module github.com/openshift-kni/cnf-features-deploy
 //   - openshift-ci/Dockerfile*
 //   - ztp/resource-generator/Containerfile
 //   - ztp/tools/pgt2acmpg/go.mod
-go 1.22
-toolchain go1.22.5
+go 1.22.4
+
+toolchain go1.23.7
```

Comment:
I don't think this is right lets bump the all project to 1.23 not only the toolchain

Reply:
`ghw` 0.16.0 doesn't require golang 1.23, so we should probably avoid this change entirely